packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4621,6 +4786,105 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe('#injectMeetingInfo', () => {
```

Comment:
suggestion: Theoretically there could be a delayed fetch pending and `injectMeetingInfo` could be called during this time, so it might be worth checking that `fetchMeetingInfoTimeoutId` gets cleared when this is the case.